Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -259,13 +278,15 @@ spec:
 
 ### Erasure Coding
 
-[Erasure coding](http://docs.ceph.com/docs/master/rados/operations/erasure-code/) allows you to keep your data safe while reducing the storage overhead. Instead of creating multiple replicas of the data,
-erasure coding divides the original data into chunks of equal size, then generates extra chunks of that same size for redundancy.
+[Erasure coding](http://docs.ceph.com/docs/master/rados/operations/erasure-code/) allows you to keep your data safe while reducing the storage overhead. Instead of creating multiple full replicas of the data,
+erasure coding (EC) divides the original data into chunks of equal size, then generates additional chunks of that size for redundancy.  This is akin to RAID5 or RAID6 but much more flexible.
 
-For example, if you have an object of size 2MB, the simplest erasure coding with two data chunks would divide the object into two chunks of size 1MB each (data chunks). One more chunk (coding chunk) of size 1MB will be generated. In total, 3MB will be stored in the cluster. The object will be able to suffer the loss of any one of the chunks and still be able to reconstruct the original object.
+For example, if you have a RADOS object of size 2MB, the simplest erasure coding with two data chunks would divide the object into two chunks of size 1MB each (data chunks). One more chunk (coding chunk) of size 1MB will be generated. In total, 3MB will be stored in the cluster. The object will be able to suffer the loss of any one of the chunks and still be able to reconstruct the original object.
 
-The number of data and coding chunks you choose will depend on your resiliency to loss and how much storage overhead is acceptable in your storage cluster.
-Here are some examples to illustrate how the number of chunks affects the storage and loss toleration.
+The number of data and coding chunks you choose will depend on your requirements for availability and data loss protection and how much storage overhead is acceptable.
+Here are some examples to illustrate how the number of chunks affects the storage and loss toleration.  As we discuss above, M=1 is not recommended for irreplaceable data.
+In most cases a value of K greater than 6 or 8 results in linear impact to write and recovery performance and demands additional failure domains, while the incremental
+increase of the usable to raw capacities exhibits diminishing returns.
 
 | Data chunks (k) | Coding chunks (m) | Total storage | Losses Tolerated | OSDs required |
 | --------------- | ----------------- | ------------- | ---------------- | ------------- |
```

Comment:
While you're at it, want to remove the 2+1 row from this table since it's really not recommended?

Reply:
Love to, had left it out of deference.  Pushing.